Autopilot/SensorFusion/Inc/sensorFusionInterface.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef SENSORFUSIONINTERFACE_H
+#define SENSORFUSIONINTERFACE_H
```

Comment:
For these header names, I would rather you separate words with underscores. So this becomes `SENSOR_FUSION_INTERFACE_H`. It just makes everything easier to read :))

Reply:
sounds good, will do